apiserver/facades/controller/undertaker/undertaker_test.go

```diff
@@ -79,14 +83,30 @@ func (s *undertakerSuite) TestNoPerms(c *gc.C) {
 			nil,
 			nil,
 			nil,
+			nil,
 		)
 		c.Assert(err, gc.ErrorMatches, "permission denied")
 	}
 }
 
 func (s *undertakerSuite) TestModelInfo(c *gc.C) {
+	name, err := user.NewName("user-admin")
+	c.Assert(err, jc.ErrorIsNil)
+
 	otherSt, hostedAPI, _ := s.setupStateAndAPI(c, false, "hostedmodel")
+	s.mockModelInfoService.EXPECT().GetModelInfo(gomock.Any()).Return(coremodel.ModelInfo{
+		UUID:            "9d3d3b19-2b0c-4a3f-acde-0b1645586a72",
+		Name:            "hostedmodel",
+		CredentialOwner: name,
+	}, nil).Times(2)
+
 	st, api, _ := s.setupStateAndAPI(c, true, "admin")
+	s.mockModelInfoService.EXPECT().GetModelInfo(gomock.Any()).Return(coremodel.ModelInfo{
+		UUID:            "9d3d3b19-2b0c-4a3f-acde-0b1645586a72",
```

Comment:
suggestion: would it be better to use use modeltesting.GenModelUUID and then store this modelUUID in the struct instead of hardcode?

Reply:
thanks. fixed.